### PR TITLE
app monetization sample nodejs

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-app-monetization/nodejs/AppPackage/manifest.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "{Domain Name}}"
+  ],
+  "subscriptionOffer": {
+     "offerId": "{{Plan-id}}"  
+     },
+    "webApplicationInfo": {
+    "id": "{{App-id}}",
+    "resource": "https://RscBasedStoreApp"
+    },
+	"authorization": {
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/AppPackage/manifest.json:52]: We can align it properly

File: samples/tab-app-monetization/nodejs/ClientApp/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "tab-channel-context-app",
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/ClientApp/package.json:2]: Rename package name as per current app

File: samples/tab-app-monetization/nodejs/ClientApp/public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Website created using create-react-app"
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/ClientApp/public/index.html:9]: Can we change content as well as per the app ?

File: samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+const Tab = () => {
+
+    React.useEffect(() => {
+        app.initialize();
+    }, [])
+
+
+    var planInfo = {
+        planId: "<Plan id>", // Plan Id of the published SAAS Offer
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx:14]: How to create SAAS offer => Please add steps or link in Readme file.

File: samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        term: "<Plan Term>" // Term of the plan.
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx:15]: What value we need to add it here, please specify in readme, if not done ?

File: samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    const handlePurchaseDialog = () => {
+
```
Comment (ChetanSharma-msft) [inline samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx:19]: Remove empty extra line

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and added my comments

File: samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx:14]: Yes we have provided steps in readme for creating SAAS offer

File: samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx:15]: Yes we have added in the comments what value to be added

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**